revolv/project/models.py

```diff
@@ -426,6 +455,34 @@ def categories(self):
         return [category.title for category in self.category_set.all()]
 
     @property
+    def updates(self):
+        """
+        :return: The set of all ProjectUpdate models associated with this project.
+        """
+        #return self.update_set.all()
+        return self.update.all()
+
+    def add_update(self, text):
+        update = ProjectUpdate(update_text=text, project=self)
+        update.save()
+
+class ProjectUpdate(models.Model):
+    update_text = models.TextField(
+        'Update text',
+        help_text = "What should the update say?"
+    )
+    
+    date = models.DateField(
+        'Date of update creation',
+        help_text = "What time was your update created?",
+        auto_now_add = True
+    )
+    
+    project = models.ForeignKey(
+        Project,
+        related_name="update"
```

Comment:
I think "updates" might be a better related name, `project.update.all()` is a little weird